lib/faker/date.ex

```diff
@@ -47,15 +45,26 @@ if Version.match?(System.version(), ">= 1.3.0") do
     """
     @spec forward(integer) :: Date.t
     def forward(days) do
-      unix_now =
-        DateTime.utc_now()
-        |> DateTime.to_unix()
+      Faker.DateTime.forward(days)
+      |> DateTime.to_date
+    end
+
+    @doc """
+    Returns a random date between two dates, start date and end date not
+    included
+    """
+    @spec between(Date.t, Date.t) :: Date.t
+    def between(from, to) do
+      Faker.DateTime.between(to_datetime(from), to_datetime(to))
+      |> DateTime.to_date
+    end
 
-      sign = if days < 0, do: -1, else: 1
+    # private
```

Comment:
This is a convention thing that helps me visually separate between public and private functions. I think most people would understand that `# private` doesn't have a special meaning within Elixir. I'm happy to abide by whatever gets this PR merged.